app/views/imported_clients/_table.haml

```diff
@@ -21,7 +21,7 @@
                 = link_to({action: :edit, id: imported_client}) do
                   = imported_client.first_name
               %td= checkmark imported_client.family_member
-              %td= imported_client.days_homeless_in_the_last_three_years
+              %td= imported_client.current_assessment.days_homeless_in_the_last_three_years
```

Comment:
Is it worth setting up delegation for all of these things that used to exist on the imported client.  Delegate to the current_assessment.